code4romania/stam-acasa
# Unselect other options when a `mutuallyExclusive` option is selected

### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Closes https://github.com/code4romania/stam-acasa/issues/279

Please mention the main changes this PR brings.

### How has it been tested?

Manually tested



> When 'Niciuna de mai sus' is selected, you should not be able to select other option.

regarding this i think it's better UX to allow users to select other options, but automatically deselect the `mutuallyExclusive` options instead of forcing user to figure out and unselect it. if other options are selected and then a `mutuallyExclusive` option is selected, the others will automatically unselect.
